public/data/Kendall_Does_Coding.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Abdus Salam",
```

I think the name here should be Kendall Pinto 😅 am I right?
```suggestion
  "name": "Kendal Pinto",
```

---

Lol, I just took it from his template and did the needed edits LOL

---

Thanks